views/partials/social_media_sharing_buttons.hbs

```diff
@@ -0,0 +1,13 @@
+<ul class="social-media-sharing-buttons">
+  <li>
+    SOCIAL MEDIA PLACEHOLDERS : 
```

Comment:
Nit: not sure if we need this label. It doesn't add much from a UI/a11y perspective, and looked kind of funny in @groovecoder's screenshot.

Reply:
@groovecoder  very strange! nightly nightly? or dark moon nightly? I can't recreate.